src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+# Root logger option
```

Comment:
I'm going to go with it is okay. Any sane person will configure the logger on their app. I've certainly appreciated having logs their to examine.

Reply:
I set up the properties file so that logs go to both stdout as well as a file.  The top line is where the developer can change the directive:
`log4j.rootLogger=DEBUG, file, stdout` <-- take out `file` to ensure it doesn't go into a file. 

But either way, yes run it by your peers. I really added it in there so that I could do some debugging on my end as I worked through your API.

Reply:
It seems fine to me to keep it there. Thanks again! The more people that use the library the more bugs we can find and fix. :)